product_configurator_sale/tests/test_sale_order_line.py

```diff
@@ -0,0 +1,119 @@
+#  Copyright 2024 Simone Rubino - Aion Tech
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import tests
+from odoo.fields import first
+from odoo.tests import Form
+
+
+class TestSaleOrderLine(tests.TransactionCase):
```

Comment:
Can you please use `BaseCase` to turn of tracking? As an alternative, turn it off explicitly.
Thanks :)

Reply:
Thanks for having a look!
Yes, this is something I discovered only recently so this old code doesn't have it; added inheritance from `BaseCommon`.